libraries/commerce/favorites/subscriptions/index.js

```diff
@@ -53,4 +55,26 @@ export default function favorites(subscribe) {
       }
     });
   });
+
+  subscribe(addRemoveBufferedFavorites$, (bufferedParams) => {
+    if (!(Array.isArray(bufferedParams) && bufferedParams.length > 0)) {
```

Comment:
this will resolve to true in two cases. 1) bufferedParams is not an array. 2) bufferedParams is an array but it has a length less than 1.  I concede that bufferedParams will almost never not be an array. In fact if RXJS's buffer operator is to be trusted it will always return an array. Perhaps I am being paranoid with the isArray check but there will be times that bufferedParams is an empty array. For instance, If the requestSync (formerly known as requestForceFavoritesBufferClear) function is called by some outside extension there may be no buffered addFavorites or removeFavorites actions buffered.